### PR TITLE
COCO Pretrain for VOC and Adding DeepLabV3

### PR DESCRIPTION
**Pull request by zhanghang1989**

### CODE REVIEW

File: gluoncv/loss.py [hunk 1]
```diff
@@ -294,24 +295,53 @@ class SoftmaxCrossEntropyLossWithAux(SoftmaxCrossEntropyLoss):
     ignore_label : int, default -1
         The label to ignore.
     """
-    def __init__(self, aux=True, aux_weight=0.2, ignore_label=-1, **kwargs):
+    def __init__(self, aux=True, mixup=False, aux_weight=0.2, ignore_label=-1, **kwargs):
```
Comment (hetong007) [inline gluoncv/loss.py]: Propose to change the name of the class, since 'Aux' means two losses, while mixup means two labels. 

Also preferrably incorporate label smoothing into the loss: https://github.com/hetong007/gluon-cv/blob/master/scripts/classification/imagenet/train_imagenet.py#L266

**Comment by zhanghang1989** [comment]
addressing https://github.com/dmlc/gluon-cv/issues/150

File: docs/model_zoo/index.rst [hunk 2]
```diff
@@ -256,6 +256,7 @@ Table of pre-trained models for semantic segmentation and their performance.
 | fcn_resnet50_ade  | FCN [6]_     | 77.1      | 38.5      | `shell script <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/segmentation/fcn_resnet50_ade.sh>`_      | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/segmentation/fcn_resnet50_ade.log>`_      |
 +-------------------+--------------+-----------+-----------+-----------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
 | psp_resnet50_ade  | PSP [12]_    | 78.4      | 41.1      | `shell script <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/segmentation/psp_resnet50_ade.sh>`_      | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/segmentation/psp_resnet50_ade.log>`_      |
+>>>>>>> upstream/master
```
Comment (hetong007) [inline docs/model_zoo/index.rst]: Please fix the format

**Comment by mli** [comment]
Job PR-241-28 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-241/28/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-241-29 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-241/29/index.html 
Code coverage of this PR:  vs. Master:
